src/components/OnBoarding/ThirdForm.jsx

```diff
@@ -0,0 +1,329 @@
+import { useState, useEffect } from 'react';
+import propTypes from 'prop-types';
+import {
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalHeader,
+  ModalBody,
+  Button,
+  Checkbox,
+  Text,
+  Box,
+  Flex,
+  FormLabel,
+  FormControl,
+  Input,
+  Stack,
+  Select,
+  Alert,
+  AlertIcon,
+  Link,
+  AlertTitle,
+  HStack,
+  Image,
+  Textarea,
+  ModalCloseButton,
+} from '@chakra-ui/react';
+import FPH_LOGO from './fph_logo.png';
+
+const ThirdForm = ({ formData, handleChange, prevStep, handleSubmit, setFormData }) => {
+  const [businessHours, setBusinessHours] = useState('');
+  useEffect(() => {
+    setFormData({ ...formData, businessHours });
+  }, [businessHours]);
```

Comment:
⚠️ **[eslint]** <[react-hooks/exhaustive-deps](https://github.com/facebook/react/issues/14920)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>React Hook useEffect has missing dependencies: 'formData' and 'setFormData'. Either include them or remove the dependency array. If 'setFormData' changes too often, find the parent component that defines it and wrap that definition in useCallback.